packages/server/migrations/201908021535001-template-joined-to-community.js

```diff
@@ -0,0 +1,46 @@
+const { TEMPLATE_NAME } = require('../src/models/enums/template-name');
+const { FeedItem } = require('../src/models/classes');
+
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.sequelize.query(`BEGIN`);
+
+      const template = new FeedItem({
+        item: '',
+        description: '{{ agent.name }} joined to community',
```

Comment:
What is the reason to have these descriptions be in the database, as opposed to direct in the code? They seem harder to change this way.

Reply:
Is this because it makes these editable by admins or similar?

Reply:
Yes, exactly